MIP61/MIP61.md

```diff
@@ -77,6 +78,10 @@ Vote outcomes are largely determined by how well voters are informed. As Recogni
 
 The intention behind the Performance Modifier is that these parameters be considered over the duration of time that a Recognized Delegate is active. The advantage of such an approach is that it allows Recognized Delegates that have been performing at high levels flexibility in terms of unexpected ability to vote, for example, in the event of illness, or planned inactivity, such as a vacation period. If a Recognized Delegate's parameters were calculated on a rolling-basis, for example over three months, taking a week off would have an outsized effect on compensation, particularly if it was a week with a large number of Governance Polls. If it is difficult or uneconomical for Recognized Delegates to take time off when required, there is potential this could lead to burnout and this is consequently undesirable.
 
+### Abstain Modifier Motivation
+
+The intention behind the Abstain Modifier is to avoid the problem of getting compensated even if a delegate is constantly just abstaining on polls. To a certain extend it can be expected that delegates take a position on the governance questions.
```

Comment:
Suggested slight rewording, grammar, and clarification

```suggestion
The intention behind the Abstain Modifier is to avoid the problem of getting compensated even if a delegate is constantly just abstaining on polls. To a certain extent, Recognized Delegates should be expected to take a position on governance questions and Executive Votes. However, it is recognized that in certain situations, Recognized Delegates may have good reasons to abstain from a specific vote.
```

Reply:
i mean, this does not really apply to executives right? i think it is perfectly okay to support no executive if proper reasoning is provided.

Reply:
Agree, but IMO RDs should still be taking a position, which could be abstain.

Reply:
abstain on a executive? how is that different from not supporting it?

![image](https://user-images.githubusercontent.com/6298245/199974782-8e27407f-1cb9-45e9-801c-f18140b8ed39.png)

this is why i am asking, i would say this is all about polls

Reply:
So *technically* there's a difference between abstaining and not supporting. At the smart contract level it's the same result (unless someone moves MKR to make it more difficult for an exec to get the hat). But at the social level there is a distinct difference, no? Ultimately it's not a particularly big deal so feel free to word it however you like 😄

Reply:
i guess i gonna reword it to just catch polls and not execs. 

execs still feel pretty binary to me
- you either support it by adding your MKR to it withing 72h
- you do not support it by leaving your MKR on the previous hat (which is likely the most powerful move to boycott the new spell) - and communicate your reasons within 72h